cmake/ecbuild_get_test_data.cmake

```diff
@@ -60,10 +60,17 @@ function( _download_test_data _p_NAME _p_DIR_URL _p_DIRLOCAL _p_CHECK_FILE_EXIST
 
       add_custom_command( OUTPUT ${_p_NAME}
         COMMENT "(curl) downloading ${_p_DIR_URL}/${_p_NAME}"
-        COMMAND ${CURL_PROGRAM} --silent --show-error --fail --output ${_p_DIRLOCAL}/${_p_NAME}
-                --retry ${ECBUILD_DOWNLOAD_RETRIES}
-                --connect-timeout ${ECBUILD_DOWNLOAD_TIMEOUT}
-                ${_p_DIR_URL}/${_p_NAME} )
+        if( ${_p_INSECURE_CURL} )
+          COMMAND ${CURL_PROGRAM} --insecure --silent --show-error --fail --output ${_p_DIRLOCAL}/${_p_NAME}
+                  --retry ${ECBUILD_DOWNLOAD_RETRIES}
+                  --connect-timeout ${ECBUILD_DOWNLOAD_TIMEOUT}
+                  ${_p_DIR_URL}/${_p_NAME} )
+        else()
+          COMMAND ${CURL_PROGRAM} --silent --show-error --fail --output ${_p_DIRLOCAL}/${_p_NAME}
+                  --retry ${ECBUILD_DOWNLOAD_RETRIES}
+                  --connect-timeout ${ECBUILD_DOWNLOAD_TIMEOUT}
+                  ${_p_DIR_URL}/${_p_NAME} )
+        endif()
 
   else()
```

Comment:
Can you similarly add the `--no-check-certificate` option for wget?

Reply:
Yes, I have added the same statement as for curl:
```cmake
      if( _p_INSECURE )
        set( INSECURE_WGET "--no-check-certificate" )
      else()
        set( INSECURE_WGET "" )
      endif()
```